pkg/master/master.go

```diff
@@ -774,7 +782,85 @@ func (m *Master) api_v1() *apiserver.APIGroupVersion {
 	return version
 }
 
-func (m *Master) InstallThirdPartyAPI(rsrc *experimental.ThirdPartyResource) error {
+func (m *Master) HasThirdPartyResource(rsrc *expapi.ThirdPartyResource) (bool, error) {
```

Comment:
IMO the interface comment should describe the contract and the
implementation comment should describe what that means in specifically, but
I don't know that we have a rule.

On Thu, Sep 17, 2015 at 9:06 PM, Brendan Burns <notifications@github.com>
wrote:

> In pkg/master/master.go
> <https://github.com/kubernetes/kubernetes/pull/13822#discussion_r39822616>
> :
>
> > @@ -774,7 +782,85 @@ func (m *Master) api_v1() *apiserver.APIGroupVersion {
> >  	return version
> >  }
> >
> > -func (m *Master) InstallThirdPartyAPI(rsrc *experimental.ThirdPartyResource) error {
> > +func (m *Master) HasThirdPartyResource(rsrc *expapi.ThirdPartyResource) (bool, error) {
>
> Done, though what is our policy on Godoc vis a vis interface
> implementations. Shouldn't we defer to the comment in the interface
> specification?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13822/files#r39822616>.
>